pyproject.toml

```diff
@@ -1,7 +1,7 @@
 [tool]
 [tool.poetry]
 name = "hoymiles_modbus"
-version = "0.6.2"
+version = "0.6.3"
```

Comment:
version is bumped as a part of the release workflow. Please don't modify manually.

Reply:
I'm working with Python 3.12. Hoymiles Modbus is a library and the only sensible way of assessing if it works or not is to actually use the library. To achieve this, I'm using Hoymiles MQTT (with Python 3.12). To get both working together, a version bump is needed.

So far, I have managed to get Hoymiles Modbus working with latest PyModbus (https://github.com/HQJaTu/hoymiles_modbus/tree/feat/framer-update). As none of my PRs are going forward, I don't think it makes any sense to post any further changes here.

To get things moving forward, I cannot do much. Those builds are failing or getting cancelled without any obvious reason or failure. Ultimately I have no idea how to un-tangle this intertwined mess. Maybe you should take charge of these and make Python 3.12 working and make repository and its automated builds work. This would enable me to contribute further bugfixes and improvements involving daily counter reset and MQTT topic.

Reply:
Yeah, I feel your pain. This package hasn't been updated for a long time. Many things have changed since 0.6.2 release which resulted in various issues. Eventually, I've managed to clean up the package and add support for Py 3.12, so I'm closing this PR.